client/my-sites/plans/jetpack-plans/product-grid/utils.ts

```diff
@@ -26,6 +38,13 @@ export const getPlansToDisplay = ( {
 		? [ getMonthlyPlanByYearly( currentPlanSlug ), getYearlyPlanByMonthly( currentPlanSlug ) ]
 		: [];
 
+	const oldPlansToNewPlans: { [ key: string ]: string } = {
+		[ PLAN_JETPACK_SECURITY_DAILY ]: PLAN_JETPACK_SECURITY,
+		[ PLAN_JETPACK_SECURITY_DAILY_MONTHLY ]: PLAN_JETPACK_SECURITY_MONTHLY,
+		[ PLAN_JETPACK_SECURITY_REALTIME ]: PLAN_JETPACK_SECURITY_PRO,
+		[ PLAN_JETPACK_SECURITY_REALTIME_MONTHLY ]: PLAN_JETPACK_SECURITY_PRO_MONTHLY,
+	};
```

Comment:
I'm assuming this a sort of safety net for us to lean on during the transition from old Backup/Security plans to new ones? My impression was that all existing subscriptions would be updated to point to our new product slugs once they launch, so as long as we're able to handle both sets of slugs, all should be well. Maybe I'm missing a link somewhere?

If that's the case, though, I wonder if it would make sense instead to trust the API to send us the correct product slug -- handling both old and new versions for a bit -- and then later removing the old slugs, instead of doing conversions like we see here.

Reply:
It was partially as a safety net during the transition, and partially to make it easy to test the UI before we've actually changed everything in the Store.